app/src/common/shared/org/mozilla/vrbrowser/search/suggestions/SuggestionsProvider.java

```diff
@@ -17,6 +18,8 @@
 
 public class SuggestionsProvider {
 
+    private static final String LOGTAG = SuggestionsProvider.class.getSimpleName();
```

Comment:
Is this how we want to handle LOGTAGs going forward? It makes it hard to grep for FxR generated logs.

Reply:
Android Studio already gives you the option of showing only the current app logs so I think filtering by class it's more useful when you want to debug a specific class (or group of classes if you use `|`). I'd rather start moving logs to this format.

Reply:
So the problem I have with using AndroidStudio is that there are a number of log messages generated in the same process that are not FxR log message. For example on the Focus, there is that annoying foviation log message that spams the logcat. Being able to filter on VRB was how I was able to focus on just message from our code and not underlying (and often noisy) system log message.